### ISSUE TITLE
SPARQL query: copyrighted figures in the NCA3

### ISSUE THREAD

**Issue by justgo129** [USGCRP/gcis-ontology#126]
Objective: Identify the (a) copyrighted figures which exist in the NCA3 and (b) the non-copyrighted ones.

This is the query to date:
select ?figure from http://data.globalchange.gov where {
   ?s a gcis:Report .
   ?s gcis:hasFigure ?figure .
   ?figure dcterms:rights [ rdf:value "Free to use with credit to the original figure source."^^xsd:string; ]
}

and of course we could modify the penultimate line as necessary.

Only column names show up in the API output.  Could anyone fix this query?  This is a real-life example of a request we received.  Thanks!

**Comment by zednis** [comment]
There are no statements using predicate `dcterms:rights` in the triplestore.

```
select ?figure ?rights from <http://data.globalchange.gov> where {
?figure dcterms:rights ?rights
}
```

has 0 results

**Comment by zednis** [comment]
actually, scratch my last comment.  The `dcterms` prefix must not be configured for the sparql endpoint.

This query works:

```
PREFIX dcterms: <http://purl.org/dc/terms/>

select ?figure from <http://data.globalchange.gov> where {
?s a gcis:Report .
?s gcis:hasFigure ?figure .
?figure dcterms:rights [ rdf:value "Free to use with credit to the original figure source."^^xsd:string; ]
}
```

**Comment by justgo129** [comment]
Great, @zednis .  I have subsequently restricted the query to NCA3 ones:
PREFIX dcterms: http://purl.org/dc/terms/

select ?figure from http://data.globalchange.gov where {
?s a gcis:Report .
?s dcterms:title "Climate Change Impacts in the United States: The Third National Climate Assessment"^^xsd:string .
?s gcis:hasFigure ?figure .
?figure dcterms:rights [ rdf:value "Free to use with credit to the original figure source."^^xsd:string; ]
}

which comes up with 147 figures, out of the 290 from the NCA3.

**Comment by justgo129** [comment]
Closed #126 due to successful resolution.  I'm surprised that the dcterms prefix isn't configured, though.  Anyway, thanks, @zednis!

**Comment by justgo129** [comment]
@zednis could you please add this to the test suite?  I know it's a relatively simple query but it's still worth adding.  Thanks.

**Comment by justgo129** [comment]
Closed #126 due to merged https://github.com/USGCRP/gcis-sparql/pull/5.
